index.src.html

```diff
@@ -423,6 +423,9 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   and vulnerable state. Removing specific cookies is best done via expiration 
   using the `Set-Cookie` header.
 
+  Note: If we clear the [=Accept-CH cache=] via the `Clear-Site-Data` header then any
+  `Accept-CH` and `Critical-CH` headers in the same request must be ignored.
```

Comment:
I looked around a bit to try to understand where `Accept-CH` and `Critical-CH` are parsed and cached, since this note isn't going to make things happen by fiat. https://wicg.github.io/client-hints-infrastructure/#fetch wasn't as helpful as I'd hoped. :)

Presumably you know where you're going to need to make this change? If so, LGTM.

Reply:
Yeah, this is going to require more work but I want to be sure to note the expectation somewhere.